src/components/Calculator.js

```diff
@@ -1,76 +1,163 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Calculator.css';
 import Display from './chiled';
+import calculate from '../logic/calculate';
 
 function Calculator() {
+  const [calculatorData, setCalculatorData] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
+
+  const handleClick = (buttonName) => {
+    setCalculatorData(calculate(calculatorData, buttonName));
+  };
+
```

Comment:
- [ ] Good job implanting some event features on your calculator 👏 , but some of them are not working 👍 , 
1) The user can not type two or more numbers
2) The equal sign is not working 
3) etc
Kindly fix those issue and make sure those errors are fix to go to the next step 👍 


https://github.com/kerlos-bahaa/Math-magicians/assets/80458199/932eb70e-644e-49a7-8a05-f4a8b24a13ae